From 5a055b561f5403a1a8e30d2d614cf51293aec5a2 Mon Sep 17 00:00:00 2001
From: Rob Browning <rlb@defaultvalue.org>
Date: Sun, 17 Apr 2011 12:26:59 -0500
Subject: Add i386 to cpp_undefs in configure.in to fix i386 multiarch builds.

Multiarch builds should now work on i386.

Multiarch builds failed on i386 because the toolchain triplet does not
match the multiarch directory (i686 != i386).

Author: Rob Browning <rlb@defaultvalue.org>
Date: Mon, 31 Jan 2011 01:13:35 +0100
Applied-By: Rob Browning <rlb@defaultvalue.org>
Forwarded: bug-gnu-emacs@gnu.org
Closes: #620795
---
 configure.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.in b/configure.in
index 0639715..94efa35 100644
--- a/configure.in
+++ b/configure.in
@@ -3099,7 +3099,7 @@ test "${exec_prefix}" != NONE &&
 # the C preprocessor to some helpful value like 1, or maybe the empty
 # string.  Needless to say consequent macro substitutions are less
 # than conducive to the makefile finding the correct directory.
-[cpp_undefs="`echo $srcdir $configuration $canonical unix |
+[cpp_undefs="`echo $srcdir $configuration $canonical unix i386 |
   sed -e 's/[^a-zA-Z0-9_]/ /g' -e 's/^/ /' -e 's/  *$//' \
   -e 's/  */ -U/g' -e 's/-U[0-9][^ ]*//g'`"]
 
-- 
1.7.6.3

